### DISCUSSION TITLE
Eviction of disk not doing anything

### DISCUSSION THREAD

**Discussion by vinid223** [longhorn/longhorn#2993] [Q&A] [answered]
Hello,

I need some help. I've added disks to my nodes to replace the current smaller disks. When I set the old disks on each node to enable eviction as well as disable scheduling, nothing is happening. 

I can't see any event, logs or anything in longhorn UI that can tell me what is going on.

Where could I look for logs in any pods ? Is there a special state I need to set the pod to detach the volumes ?

For infos, the new disks can accept replicas when I force delete a replicat or change the replicat amount. They are formatted for the ext4 FS.

Thank you

**Comment by PhanLe1010** [comment, marked as answer]
I saw the issue you created! It is a bug on our end. We will fix it. Thank you for reporting

**Reply by jenting** [reply]
> I saw the issue you created! It is a bug on our end. We will fix it. Thank you for reporting

Link to #2995
